assets/js/theme/global/modal.js

```diff
@@ -204,6 +206,8 @@ export class Modal {
             this.focusTrap = focusTrap.createFocusTrap(this.$modal[0], {
                 escapeDeactivates: false,
                 returnFocusOnDeactivate: false,
+                allowOutsideClick: true,
```

Comment:
Question: why allowOutsideClick has been added?

Reply:
By default focusTrap do not allow outside click, but our modal has such functionality. allowOutsideClick was added to allow user close modal by clicking outside of it